docs/upgrade.html

```diff
@@ -27,6 +27,14 @@ <h5><a id="upgrade_270_notable" href="#upgrade_270_notable">Notable changes in 2
         <code>default.api.timeout.ms</code>, and Kafka Streams' new <code>task.timeout.ms</code> parameters instead.
         Note that parameter <code>retry.backoff.ms</code> is not impacted by this change.
     </li>
+    <li>Altering non-reconfigurable configs of existent listeners causes <code>InvalidRequestException</code>.
+        By contrast, the previous behavior would have caused the updated configuration to be persisted, but it wouldn't
+        take effect until the broker was restarted. This change breaks behavior compatibility but the old behavior is not
```

Comment:
nit: can we take out the sentence about behavior compatibility. I think it is enough to say that the previous behavior was unintended.

Reply:
I had deleted the sentence about behavior compatibility :)